### ISSUE TITLE
QUES: Is there any update plan in the project ?

### ISSUE THREAD

**Issue by Tonysmark** [vuejs/vue-class-component#604]

**Comment by corwin-of-amber** [comment]
Please do not bury this project 😭 There is no alternative, and there will probably not be one, because all Google searches about using ES classes with Vue (2 and 3) lead to vue-class-component. If it dies, class support dies with it, and ultimately devs will switch to React.

**Comment by chubbard** [comment]
I'm here because Angularjs kicked the bucket finally, and I needed to find a new toolkit for building apps.  I've used Angular in the past and enjoyed using Typescript and class based development for frontend work.  I decided to look outside of Angular because of the Angular's troubled past, and wanted something smaller.  I was hoping Vuejs would be that, but with the lack of a plan to update vue-class-component I wonder if I'm going to be left again high and dry by yet another abandoned project which by the day seems to be the situation here too.  Do I rewrite everything back is ES and just swallow the kool-aid on the abysmal composition API?  It was fine to not have updated docs a year ago, but we're past that year mark and still docs haven't arrived.  Come on what's the deal?  Just something to explain where this is going!

**Comment by rdhelms** [comment]
@corwin-of-amber @chubbard @ThreeScreenStudios I totally agree with all your comments here...currently we're moving forward with vue-facing-decorator (the package mentioned in the deprecation notice) but the struggle while we were waiting for news from these packages was real.
https://medium.com/@robert.helms1/vue-2-to-vue-3-with-class-components-cdd6530a2b2a

**Comment by corwin-of-amber** [comment]
Thank you so much @rdhelms, the prospect of a replacement is reassuring and I will try vue-facing-decorator. It does look great; perhaps a quick porting guide from vue-class-component will help disseminate its adoption?
